### ISSUE TITLE
`[readonly]` causes complaints about `readOnly`

### ISSUE THREAD

**Issue by zoechi** [dart-lang/angular_analyzer_plugin#329]
![image](https://cloud.githubusercontent.com/assets/405837/26146012/f72e65ee-3aee-11e7-99a0-3e90f971a468.png)

```dart
class StringInputComponent extends DefaultValueAccessor implements OnChanges {
  NgModel _ngModel;
  String value;

  @Input()
  String label;

  @Input()
  bool readonly = false;
  ...
```

**Comment by zoechi** [comment]
Almost allof my input elements have a `readonly` property and I get this warning now for each of them (since todays plugin update).

Dart VM version: 1.24.0-edge.6c8733dcbd0fe2b1edbec5f9138e266c9828e7f3 (Tue May 16 19:19:16 2017) on "linux_x64"

**Comment by mk13** [comment]
Ah I know the reason for this, I'll path this right away since this is a relatively quick fix.

**Comment by mk13** [comment]
Resolved now; PR #343
